pcap/pcap.go

```diff
@@ -89,6 +89,29 @@ int pcap_set_rfmon(pcap_t *p, int rfmon) {
 #define gopacket_time_secs_t time_t
 #define gopacket_time_usecs_t suseconds_t
 #endif
+int pcap_wait(pcap_t *p, int usec) {
+	fd_set fds;
+	int fd;
+	struct timeval tv;
+
+	fd = pcap_get_selectable_fd(p);
```

Comment:
If only things were that simple... :(

Breaking Windows users is a non-starter, so we need to write code that compiles for both Windows and others.  Here, this probably means splitting the implementation up into different files and using https://golang.org/pkg/go/build/ build constraints to specify Windows vs. non-Windows code.

In short, we'll probably end up with three files:  pcap.go, for common stuff.  pcap_win.go, with the windows implementation of Windows-specific functionality.  pcap_other.go with non-windows stuff.

It would have been really great if libpcap hadn't decided to do backwards-incompatible changes.